src/redux/reducers/appstate.ts

```diff
@@ -0,0 +1,30 @@
+import { createAction, createReducer } from '@reduxjs/toolkit'
+
+// Events are described by EventType
+export enum RNAppStateType {
+  Active = 'active',
+  Inactive = 'inactive',
+  Background = 'background',
+}
```

Comment:
I've found this is actually easier to implement with simple types.

`type RNAppState = 'active' | 'inactive' | 'background'`

TypeScript will then autocomplete that for you when necessary, and detects whether or not you're using the right string. It saves you from having to import the enum.